play/src/components/OpenAnswerCard.tsx

```diff
@@ -45,7 +50,8 @@ export default function OpenAnswerCard({
   const [isBadInput, setIsBadInput] = useState(false); 
   const [katexAnswer, setKatexAnswer] = useState('');
 
-  const answerType = AnswerType[answerSettings?.answerType as AnswerType] ?? AnswerType.STRING;
+  const answerType: AnswerType = answerSettings?.answerType ?? AnswerType.STRING;
```

Comment:
`answerType` has been parsed via `networking` so we don't have to deal with the enum anymore.